### DISCUSSION TITLE
无法设置代码高亮怎么办？

### DISCUSSION THREAD

**Discussion by lingxin12** [YunYouJun/hexo-theme-starry#113] [Q&A 疑难杂症] [answered]
这是我的博客：https://www.lingxinblog.cn/

本来没有设置有关代码显示的的（默认highlight开着的），代码块长的这样：
![~4R4`LO7$VSP)2FAUMU2NEJ](https://user-images.githubusercontent.com/57349676/110213780-7376c300-7edc-11eb-841c-a0ccac58b435.png)

然后检查了hexo的版本
![U02O{ P592HLV2$6}4_}BMT](https://user-images.githubusercontent.com/57349676/110213779-72de2c80-7edc-11eb-8c2b-7ebacfb1e832.png)

于是决定配置PrismJS

这是hexo站点下的配置文件：
![TOH8PL8SS3IFC_MO}`)5R$B](https://user-images.githubusercontent.com/57349676/110213819-aa4cd900-7edc-11eb-8322-3bb4d91dc516.png)
![_{5VF8)Z)ATDD{YBUT1GKFR](https://user-images.githubusercontent.com/57349676/110213821-ad47c980-7edc-11eb-82fc-b46e4141f68d.png)

这是主题配置文件的设置：
![GNL2H@JP K}@ LM{67}WOW](https://user-images.githubusercontent.com/57349676/110213831-b8025e80-7edc-11eb-9acb-dba20ce7c49e.png)

然后发现代码块还是原来那样没有变。。。。。

大概看了一下用了同一个主题一位网友他的是这样的：
![C_L7}9RK`KT7LN$F@~L{JJW](https://user-images.githubusercontent.com/57349676/110213862-d6685a00-7edc-11eb-9f5a-b9f5ef947f74.png)

为什么我的代码无法高亮。。。（哭

**Comment by YunYouJun** [comment, marked as answer]
```sh
hexo clean
hexo s
```

**Reply by lingxin12** [reply]
感谢感谢，已经解决😁

**Reply by lingxin12** [reply]
好像又有问题了233333333
设置和上面的一样，代码变成了这样：
![VGQAO3$KL8V7GH$(LN}XCU1](https://user-images.githubusercontent.com/57349676/110214975-3ca3ab80-7ee2-11eb-8442-a402fe26f32e.png)
样式变了可是还是没有亮起来，这是我的插件，是不是少了什么渲染器之类的。。
![~0O3KV{(G6$L8{J7LNNEYPY](https://user-images.githubusercontent.com/57349676/110215023-85f3fb00-7ee2-11eb-9a9b-efb01731251e.png)

**Reply by YunYouJun** [reply]
```cpp
int main() {

}
```

![image](https://user-images.githubusercontent.com/25154432/110215073-c6537900-7ee2-11eb-8898-6835aabe3b4a.png)

**Reply by lingxin12** [reply]
感谢感谢，我把typora中代码块的c++改成cpp就好了😥
